src/utils/wallet.js

```diff
@@ -438,6 +439,8 @@ class Wallet {
         });
 
         await contract.claim({ account_id: accountId }, LINKDROP_GAS);
+
+        await store.dispatch(finishLinkdropClaim());
```

Comment:
I know that we're already dispatching using `store` in this file, so this wasn't obvious, but we're trying to extricate Redux awareness from `wallet.js`.  Instead of dispatching here, can you please enhance the `CLAIM_LINKDROP_TO_ACCOUNT` action in `actions/account.js` to call this wallet method, then dispatch this action instead of having this wallet method dispatch the action?

Reply:
sure. I have updated the action `CLAIM_LINKDROP_TO_ACCOUNT `

Reply:
:+1: thanks so much